docs/en-us/containerrecreaterequest.md

```diff
@@ -0,0 +1,108 @@
+---
+title: ContainerRecreateRequest
+---
+# ContainerRecreateRequest
+
+**FEATURE STATE:** Kruise v0.9.0
+
+ContainerRecreateRequest provides a way to let users **restart/recreate** one or more containers in an existing Pod.
+
+Just like the in-place update provided in Kruise, during a container being recreating, the other containers in the same Pod are still running.
+After the recreation completed, nothing changes in the Pod except the container's restartCount will be increased.
```

Comment:
After the recreation completed, nothing changes in the Pod except the container's restartCount will be increased.
> 
Once the recreation is completed, nothing changes in the Pod except that the recreated container's restartCount is increased.